src/DIRAC/ConfigurationSystem/Client/ConfigurationClient.py

```diff
@@ -51,7 +51,7 @@ def commitNewData(self, sData):
 
         :param sData: Data to commit, you may call this method with CSAPI and Modificator
         """
-        return self.executeRPC("commitNewData", b64encode(sData))
+        return self.executeRPC("commitNewData", b64encode(sData) if six.PY2 else sData)
```

Comment:
I do not quite understand the question, this condition when using python3 relies on JEncode when processing bytes, otherwise everything will work as before

Reply:
If I am using python2, you will call `self.executeRPC("commitNewData", b64encode(sData))`, correct ? 
The server side will receive this `b64enocded` data. Where is it decoded ?

Reply:
here https://github.com/DIRACGrid/DIRAC/pull/5811/files#diff-8ac68fb9299e87707b5337b41377c164fa1c27b15ebe89a4ef191ed6dc231f43R88